app/controllers/supervisor/notes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Supervisor::NotesController < Supervisor::BaseController
+
+  def update
+    notepad = @current_user.notepad
```

Comment:
It's better to access the `@current_user` w/o the @, ie. via its corresponding helper method.